doc/ssh_config/ssh_config.md

```diff
@@ -161,6 +175,20 @@ module sonic-ssh-server {
                         }
                     }
 				}
+				leaf auto_logout {
+					description "inactivity timeoout (min unit)";
+					default 15;
+					type uint32 {
+						range 0..35000;
+					}
+				}
+				leaf max_syslogins {
+					description "limit of concurrent system logins";
+					default 100;
+					type uint32 {

```

Comment:
uint32 -> uint8?

Reply:
I prefer to leave it as 32, to not limit it in case we will extend the upper limit in future.